002_simulation_structure/002_simulation_structure.tex

```diff
@@ -100,6 +100,209 @@ \subsection{Infrastructure}
 Base agent contains the core information of an agent, such as an agent's HP and floor. Base agent also contains the \lstinline$hpDecay()$ function as well as a function to calculate an individual agent's utility. 
 Custom agents are designed by each of the agent teams, where each one of them has a different strategy.
 
+%%%%%%%%
+%%%%%%%% Message Passing
+%%%%%%%%
+\section{Message Passing}\label{message_passing}
+\subsection{Primitive Messaging}
+An important role of Team 1 was to design a mechanism for agents to talk amongst each other in the simulation that would allow automated communication. The first step in this process was to design a system which would allow an agent to pass along a minimum viable message to their neighbours. These were the goals of communication in the MVP:
```

Comment:
```suggestion
An important role of Team 1 was to design a mechanism for agents to talk amongst each other in the simulation. The first step in this process was to design a system which would allow an agent to pass along a minimum viable message to their neighbours. These were the goals of communication in the MVP:
```